interest

Thanachot-W opened the pull request.

Thanachot-W committed:
interest

FC4RICA commented on app/src/main/java/com/fc4rica/bonbaan/ui/onboarding/FilterChip.kt:
Using theme color instead of hard coded color value. For example MaterialTheme.colorScheme.primary for background and MaterialTheme.colorScheme.onPrimary for text.

FC4RICA commented on app/src/main/java/com/fc4rica/bonbaan/ui/onboarding/OnboardingHeader.kt:
Using theme typography instead of hard coded font style. For example MaterialTheme.typography.titleSmall for title and MaterialTheme.typography.bodyMedium for subtitle.

FC4RICA commented on app/src/main/java/com/fc4rica/bonbaan/ui/onboarding/InterestScreen.kt:
```diff
@@ -0,0 +1,65 @@
+
+        Spacer(modifier = Modifier.weight(1f))
+
+        BonBaanButton(
+            text = "ตกลง",
+            onClick = { navController.navigate("nextScreen") },
+            variant = ButtonVariant.PRIMARY,
+            modifier = Modifier.fillMaxWidth()
+        )
```
Would be better if you have condition to select at least one or button would be disable

FC4RICA reviewed (changes requested):
Solid work. Just move FilterChip into components folder and you don't have to separate OnboardingHeader and InterestSelection into files. You can put those in InterestScreen, since it would be used only once. But that's a good idea. I might do the same with register pages.

Thanachot-W committed:
feature/interest

-use MaterialTheme.colorScheme
-use MaterialTheme.typography

Thanachot-W committed:
feature/interest

- move FilterChip to Component Folder
- merge Interestelection and Onboardingheader into InterestScreen
